keyboards/palette1202/lib/oled_helper.h

```diff
@@ -0,0 +1,7 @@
+#ifdef OLED_DRIVER_ENABLE
+#pragma once
```

Comment:
The `#pragma once` statement should go first.

```suggestion
#pragma once
#ifdef OLED_DRIVER_ENABLE
```